sources.cfg

```diff
@@ -4,3 +4,6 @@ extensions = mr.developer
 
 auto-checkout =
     ftw.publisher.core
+
+[branches]
+ftw.publisher.core = dj_abbreviate_constrained_types_when_poss
```

Comment:
Do we have circular dependencies (and need to set your branch here so that tests pass), or could we wait for the branch of `ftw.publisher.core` to be merged?

Reply:
Not circular deps, but sure - I will remove this, but I needed it for development and proving tests pass for review, right?